web/packages/teleterm/package.json

```diff
@@ -10,11 +10,9 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "yarn build-native-deps && yarn start-skip-native-deps",
-    "start-skip-native-deps": "electron-vite dev",
+    "start": "electron-vite dev",
     "start-electron": "electron build/app/dist/main/main.js",
     "build": "electron-vite build",
-    "build-native-deps": "electron-builder install-app-deps",
```

Comment:
Not depending on nan and not having to rebuild binaries each time that the Electron version changes is huge.

I don't have the time to thoroughly review this today, but what I think the switch to Node-API means is that node-pty is not bound to a particular Node version but instead works with any Node that supports Node-API, see:

* https://github.com/microsoft/node-pty/pull/644#issuecomment-1868755800
* https://github.com/microsoft/node-pty/issues/46#issuecomment-1986897870

I suppose that's why it works without manually rebuilding the native deps.

It's fine that electron-builder rebuilds it during packaging. This way we're always sure that the node-pty version shipped with the app was built with the same Node version that's going to be used by the app.